pkg/agent/config.go

```diff
@@ -126,22 +127,19 @@ func (c *Config) Dump() (string, error) {
 	return string(d), nil
 }
 
-func (c *Config) validate() error {
+func (c *Config) validate(isVenafi bool) error {
```

Comment:
@tfadeyi for clarity should we change this parameter name to 'isVenafiCloudMode'?

Reply:
I thought that but it doesn't sound well with name of the function `validate`. I can change it

Reply:
Thanks @tfadeyi, we can always change it again. I think when we may have to make changes again at some point when we fully transition.